src/angular/schematics/ng-add/data/css-selectors.ts

```diff
@@ -16,4 +16,23 @@ export interface SbbAngularCssSelectorData {
   };
 }
 
-export const cssSelectors: VersionChanges<SbbAngularCssSelectorData> = {};
+export const cssSelectors: VersionChanges<SbbAngularCssSelectorData> = {
+  ['version 13' as TargetVersion]: [
```

Comment:
why version 13? I'd use 'merge' like the other cases and I'd appreciate a 'merge'-test for that :)